Sources/MapboxCoreNavigation/PassiveNavigationEventDetails.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import CoreLocation
+
+struct PassiveNavigationEventDetails: NavigationEventDetails {
+    let coordinate: CLLocationCoordinate2D?
+    let created = Date()
+    let sessionIdentifier = "session-id" // TODO: create global session
+    let driverMode = "freeDrive"
+    
+    var event: String?
+    var userId: String?
```

Comment:
Consider sticking to Swift naming conventions, relying on the CodingKeys property below to associate a `userId` property with `userIdentifier` in JSON.